### ISSUE TITLE
Collision/Collider Issue with latest build. 

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#4841]
I haven't touched this section of my level in probably over 6+ months. I probably run through it once a week for testing. Now after latest update invisible physics colliders have popped up in this hallway. 

![Desktop Screenshot 2023 08 03 - 03 53 53 43](https://github.com/TheGameCreators/GameGuruRepo/assets/51464873/dd0eea93-3055-452f-9fa6-7d8a84365956)

**Comment by Kitakazi** [comment]
Sure I can when I get home, it's a pretty large level with a lot of custom media though. I'd rather not post a link that can be accessed by anyone, do you have an email I can send it too?

**Comment by Kitakazi** [comment]
Ok I figured out what has happened to cause this random collision box in my level and it is bizarre. 

I decided to start removing entities starting with the newest one by one untill I figured out which entity was tied to this random collision box. Eventually I figured out it was an entity which was originally just a blood splat wall decal. The texture was custom made by me but the model/.dbo file itself is stock media, I just changed the texture. 

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/51464873/11f832d6-4995-42d0-915d-1fac54a31073)



Anyway, when I look at this model in my level, which should be a blood decal. It has the name of the entity once I clicked it the settings under the name in the object tools library changed to  that of the Structure Editor? It's like somehow this wall decal is confused and mixed up wtih a structure from the structure editor.  When I delete the wall decal and then CTRL + Z to undo the name changed to ebe583. So I believe it was spawning collision from some unknown structure named ebe583 which is not any structure I have ever made. So I have no idea where this collision data is coming from. 

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/51464873/5e8e615a-87c6-4fa2-92b2-484b0d7a469c)

Not really sure how this is even possible or where it got the data for "ebe583"
